### PR TITLE
Add in time created to bundle.json

### PR DESCRIPTION
**Pull request by jacobfoard**
One of the use cases we've had  is keeping our models up to date on a nightly basis and having alerts for when they're X hours old. So I added a field to the bundle case class and have it auto populate when the bundle is being created. I also couldn't get this to build with the `0.12.head` version of tensorflow so I updated to the latest stable version and the unit tests passed. I can revert that if need be for the time stamp.

### CODE REVIEW

**Comment by hollinwilkins** [comment]
@jacobfoard Thank you for the PR. Will get a chance to look closer at it after DataWorks summit tomorrow. Thanks for updating TensorFlow as well, did you get a chance to test it out with the latest 1.1 version?

**Comment by jacobfoard** [comment]
I've looked into it some more and it doesn't pass the tests because the TensorFlowTransformer doesn't comply to the Transfor abstract with the new getFields requirement. I made an attempt to fix it, but I'm less knowledgeable in that area. I'll post the getFields implementation I did and get some feedback and I'll add that in. I believe that is the only thing stopping the tests from passing.

Sent from my iPhone
________________________________

**Comment by hollinwilkins** [comment]
@jacobfoard Cool, take a swing at it, if it proves to be too tricky at this time, we can return a failure there for now until we have a good solution for it. Anca and I are going to be reviewing the typing system that we have for MLeap in the next couple of weeks anyways.

File: bundle-ml/src/main/scala/ml/combust/bundle/dsl/Bundle.scala [hunk 1]
```diff
@@ -104,7 +105,8 @@ object Bundle {
     apply(BundleInfo(uid = UUID.randomUUID(),
       name = name,
       format = format,
-      version = Bundle.version), root)
+      version = Bundle.version,
+      LocalDateTime.now().toString), root)
```
Comment (hollinwilkins) [inline bundle-ml/src/main/scala/ml/combust/bundle/dsl/Bundle.scala]: Does this return iso8601 formatted dates? We should ensure this is the format we are using.

Comment (hollinwilkins) [inline bundle-ml/src/main/scala/ml/combust/bundle/dsl/Bundle.scala]: Also, to be sure of the field that is being set, just do a quick:
timeCreated = LocalDateTime.now().toString

File: project/Dependencies.scala [hunk 2]
```diff
@@ -8,7 +8,7 @@ object Dependencies {
 
   val sparkVersion = "2.1.0"
   val scalaTestVersion = "3.0.0"
-  val tensorflowVersion = "0.12.head"
+  val tensorflowVersion = "1.1.0"
```
Comment (hollinwilkins) [inline project/Dependencies.scala]: This changeset should probably be in another PR

File: python/mleap/sklearn/pipeline.py [hunk 3]
```diff
@@ -126,6 +126,7 @@ def get_bundle(transformer):
           "name": transformer.name,
           "format": "json",
           "version": __version__,
+          "timeCreated": datetime.datetime.now().isoformat(),
```
Comment (hollinwilkins) [inline python/mleap/sklearn/pipeline.py]: Again, just a note to double check that this returns iso8601 formatted date strings

File: bundle-ml/src/main/scala/ml/combust/bundle/dsl/Bundle.scala (hunk 1 above)
Comment (jacobfoard) [inline bundle-ml/src/main/scala/ml/combust/bundle/dsl/Bundle.scala]: Yea they are both in the iso8601 format, and I have reflected the other two changes.

File: project/Dependencies.scala (hunk 2 above)
Comment (jacobfoard) [inline project/Dependencies.scala]: I'll open up another issue about this and put in the code I used for `getFields`
